tests/metrics/README.md

```diff
@@ -4,25 +4,19 @@
 Some tests require you to enable the debug mode of cc-oci-runtime. To enable it, please take a look at:
 https://github.com/01org/cc-oci-runtime#18running-under-docker
 
-To run the metrics, just run:
+
+### Run all metrics tests
+To run the metrics, please use Makefile rule `metrics-tests` on the top level of this repository:
 
 ```bash
-$ cd tests/metrics
-$ ./run_docker_metrics
+$ sudo make metrics-tests
```

Comment:
Added `sudo` because if user is not part of the docker group, it will not be able to run docker command successfully, but I can remove it and add a note for this. Does that sounds good?